ArcGISRuntimeSDKQt_CppSamples/Layers/RasterFunctionFile/RasterFunctionFile.cpp

```diff
@@ -95,7 +95,7 @@ void RasterFunctionFile::componentComplete()
   m_map->operationalLayers()->append(rasterLayer);
 
   // Set initial Extent
-  Envelope env = Envelope::fromJson("{\"spatialReference\": {\"latestWkid\": 3857, \"wkid\": 102100 },\"xmax\": -13591503.517810356,\"xmin\": -13606233.44023646,\"ymax\": 4982810.138527592,\"ymin\": 4971762.696708013}");
+  Envelope env = geometry_cast<Envelope>(Envelope::fromJson("{\"spatialReference\": {\"latestWkid\": 3857, \"wkid\": 102100 },\"xmax\": -13591503.517810356,\"xmin\": -13606233.44023646,\"ymax\": 4982810.138527592,\"ymin\": 4971762.696708013}"));
```

Comment:
Is it possible to change the return type of `fromJson` to return `Envelope` and not `Geometry`?

Reply:
No, that goes against the design. There is only `Geometry::fromJson`, but it is frequently called from subclasses like `Envelope::fromJson` to convey intent, although it's no different.